tests/unit/lambdas/api_server/test_area.py

```diff
@@ -179,6 +179,10 @@ def test_update_file_validation(self, mock_format_and_send_notification, mock_co
         upload_area = UploadArea(area_id)
         uploaded_file = UploadedFile(upload_area, s3object=s3obj)
         uploaded_file.create_record()
+        response = self.client.get(f"/v1/area/{area_id}/foo.json/validate", headers=self.authentication_header)
+        validation_status = response.get_json()['validation_status']
+        self.assertEqual(validation_status, "UNSCHEDULED")
```

Comment:
nit: Whoa, this is a big test. I don't know if this is the right PR for it, but it would be great if this were broken into bite sized chunks

Reply:
Thanks. This can be fixed in this PR. Will try to see how it can be broken down